homeassistant/components/pilight.py

```diff
@@ -79,6 +86,8 @@ def send_code(call):
 
         try:
             pilight_client.send_code(message_data)
+            if send_delay > 0.0:
+                time.sleep(send_delay)
```

Comment:
So I have to track when the last call took place and manage a queue for my own to calculate the timepoint for the call correctly?